src/gui/splash.c

```diff
@@ -257,7 +257,7 @@ void darktable_splash_screen_create(GtkWindow *parent_window,
   gtk_window_set_decorated(GTK_WINDOW(splash_screen), FALSE);
   gtk_widget_show_all(splash_screen);
   gtk_widget_hide(GTK_WIDGET(remaining_box));
-  gtk_window_set_keep_above(GTK_WINDOW(splash_screen), TRUE);
+//  gtk_window_set_keep_above(GTK_WINDOW(splash_screen), TRUE);
```

Comment:
Wouldn't it be better to remove instead of commenting out?

Reply:
Yes, I'll clean that up.